source/Engine.cpp

```diff
@@ -159,6 +159,26 @@ namespace {
 		Messages::Add(tag + message);
 	}
 	
+	void DrawFlares(Ship ship, DrawList *draw, bool calcTickTock, double cloak, vector<Ship::EnginePoint> enginePoints, vector<pair<Body, int>> flareSprites, double angle, uint8_t side)
```

Comment:
If anyone has a better solution than sending 8 variables to a helper function, I'm all ears.